src/Congo.WebApi/Data/ProductAccess/InsertProductHelper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Congo.WebApi.Data.Models;
+using MediatR;
+
+namespace Congo.WebApi.Data.ProductAccess
+{
+    public class InsertProductHelper : IRequestHandler<InsertProductCommand, Product>
+    {
+
+        private readonly CongoContext _dbContext;
+
+        public InsertProductHelper(CongoContext dbContext) => _dbContext = dbContext;
```

Comment:
Personal preference but keep the curly braces here because we might want to inject something else later on